godel_process_planning/src/keyence_process_planning.cpp

```diff
@@ -0,0 +1,152 @@
+#include <godel_process_planning/godel_process_planning.h>
+
+#include <ros/console.h>
+
+// descartes
+#include "descartes_trajectory/cart_trajectory_pt.h"
+#include "descartes_planner/dense_planner.h"
+
+#include "common_utils.h"
+
+// FILE LOCAL CONSTANTS
+const static std::string JOINT_TOPIC_NAME = "joint_states"; // ROS topic to subscribe to for robot state
+
+namespace godel_process_planning
+{
+
+/**
+ * @brief Translated an Eigen pose to a Descartes trajectory point appropriate for the scan process!
+ *        Mirros the function in blend_process_planning.cpp document.
+ * @param pose
+ * @param dt The upper limit of time from the previous point to achieve this one
+ * @return A descartes trajectory point encapsulating a move to this pose
+ */
+static inline descartes_core::TrajectoryPtPtr toDescartesPt(const Eigen::Affine3d& pose, double dt)
```

Comment:
I absolutely agree here. A bunch of these methods, like the `toROSTrajectory` and `toDescartePt`and all of those probably belong in a `descartes_utils` package.